### PR TITLE
fix:  change name and email address

### PR DESCRIPTION
**Pull request by YusufMavzer**

### CODE REVIEW

File: src/Public.Api/Dockerfile [hunk 1]
```diff
@@ -1,6 +1,6 @@
 FROM mcr.microsoft.com/dotnet/runtime-deps:5.0.2
 
-LABEL maintainer "agentschap Digitaal Vlaanderen <basisregisters@vlaanderen.be>"
+LABEL maintainer = "Digitaal Vlaanderen <digitaal.vlaanderen@vlaanderen.be>"
```
Comment (ArneD) [inline src/Public.Api/Dockerfile:3]: There was no `=` before. Intentional?

File: src/Public.Api/appsettings.json [hunk 2]
```diff
@@ -1,7 +1,7 @@
 {
   "Contact": {
-    "Name": "agentschap Digitaal Vlaanderen",
-    "Email": "basisregisters@vlaanderen.be"
+    "Name": "Digitaal Vlaanderen",
+    "Email": "digitaal.vlaanderen@vlaanderen.be"
```
Comment (ArneD) [inline src/Public.Api/appsettings.json:4]: I don't think this needs to be changed.

File: src/Public.Api/Dockerfile (hunk 1 above)
Comment (YusufMavzer) [reply src/Public.Api/Dockerfile:3]: my bad

File: src/Public.Api/appsettings.json (hunk 2 above)
Comment (YusufMavzer) [reply src/Public.Api/appsettings.json:4]: The email is a ticket from 
GRAR-1982

Comment (ArneD) [reply src/Public.Api/appsettings.json:4]: ok, taken discussion in ticket

File: src/Public.Api/Dockerfile (hunk 1 above)
Comment (YusufMavzer) [reply src/Public.Api/Dockerfile]: I removed the =

**Pull request merged**